src/nomad_simulations/schema_packages/properties/thermodynamics.py

```diff
@@ -0,0 +1,289 @@
+    """
+    Physical property section describing the Gibbs free energy (i.e., enthalpy - temperature * entropy) of a (sub)system.
+    """
+
+    def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
+        super().normalize(archive, logger)
+
+
+class HelmholtzFreeEnergy(BaseEnergy):
+    """
+    Physical property section describing the Gibbs free energy (i.e., internal_energy - temperature * entropy) of a (sub)system.
+    """
+
+    def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
+        super().normalize(archive, logger)
+
+
+class ChemicalPotential(BaseEnergy):
+    """
+    Free energy cost of adding or extracting a particle from a thermodynamic system.
+    """
+
+    # ! implement `iri` and `rank` as part of `m_def = Section()`
+
+    iri = 'http://fairmat-nfdi.eu/taxonomy/ChemicalPotential'
+
+    def __init__(
+        self, m_def: 'Section' = None, m_context: 'Context' = None, **kwargs
+    ) -> None:
+        super().__init__(m_def, m_context, **kwargs)
+        self.rank = []
+        self.name = self.m_def.name
+
+    def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
+        super().normalize(archive, logger)
+
+
+class HeatCapacityCV(PhysicalProperty):
+    """
+    Physical property section describing the heat capacity at constant volume for a (sub)system.
+    """
+
+    value = Quantity(
+        type=np.float64,
+        unit='joule / kelvin',
+        description="""
+        The value of the heat capacity.
+        """,
+    )
+
+    def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
+        super().normalize(archive, logger)
+
+
+class HeatCapacityCP(PhysicalProperty):
+    """
+    Physical property section describing the heat capacity at constant volume for a (sub)system.
+    """
+
+    value = Quantity(
+        type=np.float64,
+        unit='joule / kelvin',
+        description="""
+        The value of the heat capacity.
+        """,
+    )
+
+    def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
+        super().normalize(archive, logger)
+
+
+################################
+# other thermodynamic properties
+################################
+
+
+class Virial(BaseEnergy):
+    """
+    Physical property section describing the virial of a (sub)system.
+    """
+
+    def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
+        super().normalize(archive, logger)
+
+
+class Density(PhysicalProperty):
+    """
+    Physical property section describing the density of a (sub)system.
+    """
+
+    value = Quantity(
+        type=np.float64,
+        unit='kg / m ** 3',
+        description="""
+        The value of the density.
+        """,
+    )
+
+    def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
+        super().normalize(archive, logger)
+
+
+# ? I think this may fit better elsewhere
+class Hessian(PhysicalProperty):
```

Comment:
I guess just because I wasn't sure where to put it, that's why I put the comment. Suggestions?

Reply:
Problem is when starting to categorize things.

That's why in the taxonomy task force the categories are a loosely way of putting properties under certain nodes, but nothing more.

Now, if you feel you should have a module called `thermodynamics.py` that's fine, provided that these edge cases are under control. Just imagine we have the modules like this, an someone comes with a property which is at the same time thermodynamical (which btw, it could be everything...) and mechanical. Or electronic and magnetic, like some spin-orbit coupling-related property.

Reply:
I am not married to having these separate categorical modules, I just did this for organization and to ease development. I guess you have made modules for individual properties, but your implementation is a bit more complex. It would get a bit out of hand to have each of these properties separate, no? Should we just have a single module to place simple implementations of all types of properties? Or what is your suggestion?

Reply:
Ok, let's keep it like this for now, and decide along the road